tone-analyzer/v3.js

```diff
@@ -34,7 +34,7 @@ function ToneAnalyzerV3(options) {
 util.inherits(ToneAnalyzerV3, BaseService);
 ToneAnalyzerV3.prototype.name = 'tone_analyzer';
 ToneAnalyzerV3.prototype.version = 'v3';
-ToneAnalyzerV3.URL = 'https://gateway.watsonplatform.net/tone-analyzer/api';
+ToneAnalyzerV3.URL = 'https://gateway-d.watsonplatform.net/tone-analyzer/api';
```

Comment:
Thanks - had already made the change to use the url in the constructor in the demo I'm working on.  Forgot to update the node-sdk to remove the URL.